domains/olxyz.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "xyzendev",
+        "email": "adriangota2007@gmail.com"
+    },
+    "record": {
+        "CNAME": "mslb1702.up.railway.app"
```

Comment:
gives 404 page (i know railway does that, so can you give link to the hosted website?)